### ISSUE TITLE
Template: Search is not understandable to Assistive Technology

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#80]
The Search modal does not have all of the ARIA components to make it understandable to all users. 

Follow the [ARIA Pattern for Modal (Dialogs)](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/)

1.3.1 Info and Relationships

**Comment by syarwick23** [comment]
Fixed by new template
